recipes/tiny-aes-c/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.32.0"
+
+class TinyAesCConan(ConanFile):
+    name = "tiny-aes-c"
+    version = "1.0.0"
+    license = "The Unlicense"
+    homepage = "https://github.com/kokke/tiny-AES-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Small portable AES128/192/256 in C"
+    topics = ("encryption", "crypto", "AES")
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    _options_dict = {
+        # enable AES128
+        "AES128": [True, False],
+        # enable AES192
+        "AES192": [True, False],
+        # enable AES256
+        "AES256": [True, False],
+        # enable AES encryption in CBC-mode of operation
+        "CBC": [True, False],
+        # enable the basic ECB 16-byte block algorithm
+        "ECB": [True, False],
+        # enable encryption in counter-mode
+        "CTR": [True, False],
+    }
+
+    options = _options_dict
```

Comment:
I don't know. Except for fPIC, there is a consensus that all options in CCI recipes should be lowercase.

Moreover, shared and fPIC options are missing.